templates/new_service/terraform/application/application.tf

```diff
@@ -0,0 +1,37 @@
+module "application_configuration" {
+  source = "./vendor/modules/aks//aks/application_configuration"
+
+  namespace              = var.namespace
+  environment            = var.environment
+  azure_resource_prefix  = var.azure_resource_prefix
+  service_short          = var.service_short
+  config_short           = var.config_short
+  secret_key_vault_short = "app"
+
+  # Delete for non rails apps
+  is_rails_application = true
+
+  config_variables = {
+    ENVIRONMENT_NAME = var.environment
+    PGSSLMODE        = local.postgres_ssl_mode
+  }
+  secret_variables = {
+    DATABASE_URL = module.postgres.url
+  }
+}
+
+module "web_application" {
+  source = "./vendor/modules/aks//aks/application"
+
+  is_web = true
+
+  namespace    = var.namespace
+  environment  = var.environment
+  service_name = var.service_name
+
+  cluster_configuration_map  = module.cluster_data.configuration_map
+  kubernetes_config_map_name = module.application_configuration.kubernetes_config_map_name
+  kubernetes_secret_name     = module.application_configuration.kubernetes_secret_name
+
+  docker_image = "${local.docker_repository}:${var.docker_image_tag}"
```

Comment:
I don't think we could keep the repository in the Terraform code, because we end up having to duplicate it in the GitHub Actions deploy workflow, can we instead pass the full image in?